Sources/InstantSearchCore/Searcher/Hits/HitsSearcher.swift

```diff
@@ -127,15 +127,17 @@ public final class HitsSearcher: IndexSearcher<AlgoliaSearchService> {
        - apiKey: API Key
        - indexName: Name of the index in which search will be performed
        - query: Instance of Query. By default a new empty instant of Query will be created.
+       - isAutoSendingHitsViewEvents: flag defining whether the automatic hits view Insights events sending is enabled
        - requestOptions: Custom request options. Default is `nil`.
    */
   public convenience init(appID: ApplicationID,
                           apiKey: APIKey,
                           indexName: IndexName,
                           query: Query = .init(),
+                          isAutoSendingHitsViewEvents: Bool = false,
```

Comment:
is adding an argument not on the last position not a breaking change? or does that not matter due to the named arguments?

Reply:
I just checked, and it might actually be a breaking change for swift, I will just move it the last position then.